src/main/java/gov/cms/madie/models/measure/Group.java

```diff
@@ -21,6 +22,7 @@
 @AllArgsConstructor
 @ValidGroupScoringPopulation
 @ValidMeasureObservation
+@ValidGroupPopulationId
```

Comment:
can we not use @NotBlank annotations from javax.validations?

Reply:
@adongare That was my first thought as well, but using @NotBlank with @Id on Population.id field didn't work.

Reply:
I'm thinking this has something to do with @Valid vs  @Validated.

Reply:
@adongare Thanks so much for your help Ashok! I reverted the use of custom validation. Please have a look again. Thanks!